rescheduler/rescheduler.go

```diff
@@ -24,15 +24,21 @@ import (
 	"time"
 
 	ca_simulator "k8s.io/contrib/cluster-autoscaler/simulator"
+	ca_drain "k8s.io/contrib/cluster-autoscaler/utils/drain"
```

Comment:
No sure the relationship with CA, prefer not to dependent on it if possible. I'm ok with that for e2e.
For the common tools, I'd suggest to move to client-go.